controllers/drcluster_controller.go

```diff
@@ -443,6 +443,27 @@ func (u *drclusterInstance) clusterUnfence() (bool, error) {
 			u.object.Name, err)
 	}
 
+	// This happens in the below scenario.
+	// 2 clusters 1 & 2
+	// - App deployed in 1 with MetroDR protection
+	// - 1 goes down (or becomes inaccessible)
+	// - 1 is fenced off and app failover to 2
+	// - 1 recovers and cleaned up
+	// - just before 1 can be unfenced, 2 goes down
+	// - app needs to be migrated to 2 for which 2 should be fenced and 1 should be unfenced
```

Comment:
So the order is important here right? IOW, fence c2 first then unfence c1. Because if you unfence c1 first while c2 is down, then c2 will be tasked to unfenced it, and that would be too late. Is my understanding correct?